dart/lib/src/io_client.dart

```diff
@@ -8,9 +8,29 @@ import 'dart:io';
 
 import 'package:http/http.dart';
 import 'package:meta/meta.dart';
-import 'base.dart';
+
+import 'client.dart';
+import 'model.dart';
```

Comment:
Can we call it `protocol` instead of `model`?

Reply:
✅ protocol it is !